### PR TITLE
fix(TDinsight): add --offline/--download-only pair options

### PR DESCRIPTION
**Pull request by zitsen**
To use offline mode, you should first download the cache:

```sh
./TDinsight.sh --download-only
```

Once downloading succeed, copy all the files in this directory, and type:

```sh
sudo /path/to/TDinsight.sh --offline
```

It's done.

Ref: [TD-14977](https://jira.taosdata.com:18080/browse/TD-14977)

### CODE REVIEW

**Review by sangshuduo — APPROVED** [review]
LGTM

**Review by boxiaobj — APPROVED** [review]
LGTM

File: dashboards/TDinsight.sh [hunk 1]
```diff
@@ -246,16 +258,26 @@ get_latest_release() {
...
     wget -c https://github.com/taosdata/grafanaplugin/releases/download/v$TDENGINE_PLUGIN_VERSION/tdengine-datasource-$TDENGINE_PLUGIN_VERSION.zip
+  
+}
+
+install_plugin() {
+  which grafana-cli > /dev/null || (echo "Grafana has not been installed in the server, install it first."; exit 1)
+  if [ "$OFFLINE" = 0 ]; then
+    download_plugin
+  else
+    [ -s tdengine-datasource-$TDENGINE_PLUGIN_VERSION.zip ] || (echo "use offline cache, bug plugin not exist"; exit 1)
```
Comment (huolibo) [inline dashboards/TDinsight.sh]: but

**Review by huolibo — APPROVED** [review]
LGTM

**Pull request merged**